cfme/tests/configure/test_landing_page.py

```diff
@@ -148,8 +147,6 @@
     "Optimize / Bottlenecks",
     "Optimize / Planning",
     "Optimize / Utilization",
-    "Red Hat Access Insights",
-    "Networks / Load Balancers",
```

Comment:
Why are `Networks` being removed too? Is this to fix `cfme/tests/configure/test_landing_page.py`?

Reply:
good question

Reply:
Network Load Balancers are no longer in 5.11, it has been deprecated.